### DISCUSSION TITLE
Tauri showcase

### DISCUSSION THREAD

**Discussion by mitch1009** [tauri-apps/tauri#7643] [Q&A]
Do we have a place where people can showcase what they are building with tauri?  If not I would love to take on that and drive it.

**Comment by FabianLars** [comment]
Current options are:
https://github.com/tauri-apps/tauri/discussions/categories/show-and-tell
https://github.com/tauri-apps/awesome-tauri/
https://madewithtauri.com/ (hosted by a working group member)
And the #showcase channel on discord

A section on the website is currently not planned but is a possibility after the website rewrite.

**Comment by btangonan** [comment]
**Anima** — a native macOS companion for Claude Code

I built a Tauri v2 + Rust app that extends Claude Code's /buddy companion system. When Anthropic launched /buddy, I loved the idea but wanted more: per-project companions instead of one per account, a way to earn and unlock new ones, and the ability to actually use buddy observations in my Claude Code sessions.

So I built it. Every project gets its own ASCII companion (8 species, 5 rarity tiers, 80 unique combos), token usage earns in-app currency for re-rolls, and a Rust daemon polls all active sessions to surface observations about your work.

Tauri v2 + Rust backend + vanilla JS frontend. 4MB binary. 43 tests. MIT licensed. First app I've ever shipped.

https://github.com/btangonan/anima

**Comment by joyahmed** [comment]
DevGo - https://github.com/joyahmed/devgo

A cross-platform project launcher built with Tauri 2 and Rust. It lists every project on every filesystem the machine can see (local drives, each WSL distro, the Mac's disk), your GitHub repos and the servers in your ssh config, in four lanes. One keystroke opens a project in your editor, terminal, tmux session or agent; a server row opens ssh straight into tmux on the box.

Windows, WSL and macOS. MIT.
